### DISCUSSION TITLE
I'm getting an SSL error when previewing in safari

### DISCUSSION THREAD

**Discussion by bpfullmetal** [barrel/shopify-vite#151] [Q&A]
Hey, I'm running into the error: An SSL error has occurred and a secure connection to the server cannot be made. at https://localhost:5173/frontend/entrypoints/bundle.theme.css. This is a particularly messy them that I'm trying to rehab but I'd assume this plugin would work more or less the same for all themes. Has anyone seen this or have an idea of how to fix it?

**Comment by montalvomiguelo** [comment]
Hi, try removing this from your config file
https://github.com/barrel/shopify-vite/blob/2ee0f6e6402535802ac587445668390fe2d4dd08/examples/vite-shopify-example/vite.config.ts#L10
